debian/control

```diff
@@ -37,6 +37,11 @@ Description: CloudStack usage monitor
  The CloudStack usage monitor provides usage accounting across the entire cloud for
  cloud operators to charge based on usage parameters.
 
+Package: cloudstack-ui
+Architecture: all
+Depends: ${misc:Depends}
```

Comment:
Do we want users to be able to install just the cloudstack-ui package? If so, maybe we need to specify the actual dependencies needed for just running the UI.

Would that be cloudstack-common or is that not even needed?

Reply:
Yes, this is based on the feedback on the ML - people may just want the UI to be say hosted by nginx. One way was to ship archive, the other to just introduce a new pkg that installs it at a specific location (not in cloudstack-managment pkg, which gets UI by default for backward compatibility)

Reply:
<pre>Depends: ${misc:Depends}</pre>

That might generate a long list of dependencies for the cloudstack-ui package which aren't needed at all.

Therefor we might want to compile the list manually

Reply:
Will fix it